### PR TITLE
encryption added to logrotate

### PR DESCRIPTION
**Pull request by olafsarnow**
Changes:

- added crypt command(s) to env/common/files/system/logrotate.nginx
- removed line with python script to remove uwsgi entries from message log file from  env/common/files/system/logrotate.syslog
- added crypt command(s) to env/common/files/system/logrotate.syslog

### CODE REVIEW

File: env/common/files/system/logrotate.syslog [hunk 1]
```diff
@@ -13,8 +13,12 @@
...
     rotate 30
     daily
+    compress
+    compresscmd /usr/bin/gpg2
+    compressoptions --encrypt -r "Sys Admin" --homedir /root/.gnupg
+    compressext .gpg
+
     postrotate
       /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
-      /usr/local/bin/log_strip_out_uwsgi.py -r $1-`date --date="yesterday"  '+%Y%m%d'`
```
Comment (torfinnnome) [inline env/common/files/system/logrotate.syslog:18]: If this script is not being used, it should also be removed from the task-playbook? https://github.com/usegalaxy-no/infrastructure-playbook/blob/fc71c6a7898e13128b8e685944e66bc44447a213/env/common/tasks/logrotate.yml#L65

Comment (olafsarnow) [reply env/common/files/system/logrotate.syslog:18]: yes, you're right. I wasn't sure if it was the only use case for this script. I'll remove it tomorrow and re-request the merge.

**Comment by torfinnnome** [comment]
I'm not sure either. :-)

tir. 13. apr. 2021 kl. 21:38 skrev Olaf Sarnow ***@***.***>:

**Comment by olafsarnow** [comment]
I looked for the file in  all yml files in the  repository and didn't find it on any other place, so I have removed it from the task and files folder.

**Pull request merged**